columnflow/tasks/framework/mixins.py

```diff
@@ -686,6 +686,21 @@ def __init__(self, *args, **kwargs):
             configs=list(self.configs),
         )
 
+    def store_parts(self) -> law.util.InsertableDict:
+        parts = super().store_parts()
+        # since MLTraining is no CalibratorsMixin, SelectorMixin, ProducerMixin, ConfigTask,
+        # all these parts are missing in the `store_parts`
+
+        configs_repr = "__".join(self.configs)
```

Comment:
In other places we limit such multi-string specifiers by including the number of string and computing a hash. Should we do this here as well? It would make sense if the number of configs is expected to be >3, otherwise the paths would get quite long.

Reply:
Good point! I think this is not important at the moment since nobody currently trains on multiple configs, but it should be addressed in another PR regarding #234